src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+import exceptions.DukeException;
+import exceptions.UnknownCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+/**
+ * The main class, includes logic to run the chatbot.
+ */
+public class Duke {
+    private final Ui ui;
+    private final Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Constructs a new chatbot.
+     * @param dataFilePath Path to store the task list text file.
+     */
+    public Duke(String dataFilePath) {
+        ui = new Ui();
+        storage = new Storage(dataFilePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            System.out.println(ui.getLoadingError());
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Respond to a given input message.
```

Comment:
A very minor point here (also for some other Javadoc): It might be good to write responds instead of respond
Quoted from coding standard:  In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)